Formula/go@1.9.rb

```diff
@@ -0,0 +1,95 @@
+class GoAT19 < Formula
+  desc "Go programming environment (1.9)"
+  homepage "https://golang.org"
+
+  stable do
+    url "https://dl.google.com/go/go1.9.4.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.9.4.src.tar.gz"
+    sha256 "0573a8df33168977185aa44173305e5a0450f55213600e94541604b75d46dc06"
+
+    go_version = version.to_s.split(".")[0..1].join(".")
+    resource "gotools" do
+      url "https://go.googlesource.com/tools.git",
+          :branch => "release-branch.go#{go_version}"
+    end
+  end
+
+  keg_only :versioned_formula
+
+  option "without-cgo", "Build without cgo (also disables race detector)"
```

Comment:
Analytics show that these two options are used in 0.01% of go installs, and are never ever used in versioned go formulas such as go 1.8. Let's drop the options.